fdbclient/TenantBalancerInterface.cpp

```diff
@@ -64,6 +64,10 @@ std::string TenantMovementStatus::toJson() const {
 	statusRoot.create("destinationPrefix") = printable(tenantMovementInfo.destinationPrefix);
 	statusRoot.create("movementState") =
 	    TenantBalancerInterface::movementStateToString(tenantMovementInfo.movementState);
+	if (!tenantMovementInfo.errorMessages.empty()) {
+		statusRoot.create("TenantMovementErrorMessages") =
```

Comment:
The original name here when we had only one error was `error`. Maybe now that we have a list we can change the name to `errors`.

```suggestion
		statusRoot.create("errors") =
```

Reply:
Thanks! I'm terrible at giving name